src/main/java/core/basesyntax/domain/Fruit.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.domain;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class Fruit {
+    private Operation operation;
+    private String name;
+    private int quantity;
+
+    public Fruit(String name, int quantity) {
```

Comment:
There is an empty line at the beginning of the constructor implementation. According to the checklist, the class or method implementation should not begin with an empty line. Please remove this redundant empty line.